package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/Natim/notes#readme",
   "devDependencies": {
+    "eslint": "3.19.0",
```

Comment:
If we also want to add auto-formatting using [**prettier**](http://npm.im/prettier), it looks pretty easy (for a future PR). We'll just need some fancy globs to ignore the third party **kinto-http** and **quill** libraries:

```diff
diff --git a/package.json b/package.json
index 93ed42f..86e713e 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Displays a sidebar that lets you take notes on web pages.",
   "scripts": {
+    "format": "prettier 'src/*.js' 'src/sidebar/!(kinto-http|quill*).{css,js}' --single-quote  --write",
     "lint": "eslint src",
     "fx-web-ext": "rm -fr web-ext-artifacts/ && web-ext build -s src && unzip -l web-ext-artifacts/*.zip"
   },
@@ -17,6 +18,7 @@
   },
   "homepage": "https://github.com/Natim/notes#readme",
   "devDependencies": {
+    "prettier": "1.4.4",
     "web-ext": "^1.9.1"
   }
 }
```